### PR TITLE
Hyperlinked the logo in Readme.md

### PR DESCRIPTION
**Pull request by Arkaroy147**
Referred to issue #2697 , hyperlinked the logo for easier use

### CODE REVIEW

**Review by bluestreak01 — CHANGES_REQUESTED** [review]
“a” tag is not closed

additionally this logo is present in all translated versions of the readme

**Comment by Arkaroy147** [comment]
> “a” tag is not closed
> 
> additionally this logo is present in all translated versions of the readme

I didn't understand the a tag not closed part. Anchor tag is not supposed to be closed

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div align="center">
-  <img alt="QuestDB Logo" src="https://questdb.io/img/questdb-logo-themed.svg" width="305px"/>
+  <a href="https://questdb.io/" target="blank"><img alt="QuestDB Logo" src="https://questdb.io/img/questdb-logo-themed.svg" width="305px"/>
```
Comment (bluestreak01) [inline README.md:2]:
```suggestion
  <a href="https://questdb.io/" target="blank"><img alt="QuestDB Logo" src="https://questdb.io/img/questdb-logo-themed.svg" width="305px"/></a>
```

**Review by bluestreak01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bluestreak01) [reply README.md:2]: there are plenty of examples in the same file of correct `a` usage. Yes, you can leave them open, but this is not a well-formed HTML.

image

Comment (bluestreak01) [reply README.md:2]: in the above example `<A>` doesn't have `body` so it is self-closed `<A/>`. Here is example of `<A>` having `body`:

image

Comment (Arkaroy147) [reply README.md]: Sorry for the confusion, I modified it as desired.

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**